Lib/test/test_posix.py

```diff
@@ -272,6 +272,30 @@ def test_pread(self):
         finally:
             os.close(fd)
 
+    @unittest.skipUnless(hasattr(posix, 'preadv'), "test needs posix.preadv()")
+    def test_preadv(self):
+        fd = os.open(support.TESTFN, os.O_RDWR | os.O_CREAT)
+        try:
+            os.write(fd, b'test1tt2t3')
```

Comment:
please write more than 10 bytes, to make sure that preadv2() doesn't read more than expected.